apps/hobbies-helsinki/src/domain/event/__tests__/EventPageContainer.test.tsx

```diff
@@ -149,7 +153,9 @@ it('should render info and load other events + similar events', async () => {
 
   // click show other times
   // eslint-disable-next-line testing-library/no-unnecessary-act
```

Comment:
It could be, but the EventPageContainer-component is still implemented inside the application folder, so I would say that an unit test needs to be where the actual component is.